lib/document.js

```diff
@@ -4418,22 +4418,26 @@ Document.prototype.$populated = Document.prototype.populated;
  *     doc.$assertPopulated('other path'); // throws an error
  *
  *
- * @param {String|String[]} paths
+ * @param {String|String[]} path
  * @return {Document} this
```

Comment:
maybe consider adding `values` to the jsdoc (and maybe also to the typescript types) to explain what `values` is expected to be / used for